src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -110,7 +110,7 @@ export class ApplicationsList extends React.Component<Props, State> {
                             {this.state.applications.map((app) => (
                                 <div key={app.metadata.name} className='column column-block'>
                                     <div className={`argo-table-list__row
-                                        applications-list__entry applications-list__entry--comparison-${app.status.comparisonResult.status}`
+                                        applications-list__entry applications-list__entry--comparison-${app.status.comparisonResult.status} applications-list__entry--health-${app.status.health.status}`
```

Comment:
Looks like not-synced and healthy app will have green border, is not it? 

I think we should use green color only if app is synced and healthy, red if app either unsynced or unhealthy and blue if app is synced and progressing.

Reply:
I'm relying on precedence based on order of declaration, so putting the failure classes last overrides the earlier progressing status, and putting that after the healthy status will override the healthy status.

Reply:
@alexmt https://www.w3.org/TR/2011/REC-CSS2-20110607/cascade.html#cascading-order

> 4. Finally, sort by order specified: if two declarations have the same weight, origin and specificity, the latter specified wins. Declarations in imported style sheets are considered to be before any declarations in the style sheet itself.

Reply:
Thank you for explaining! Lgtm